auth-api/tests/unit/api/test_invitation.py

```diff
@@ -0,0 +1,203 @@
+        'sentDate': '2019-09-09',
+        'membership': [
+            {
+                'membershipType': 'MEMBER',
+                'orgId': str(org_id)
+            }
+        ]
+    }
+    rv = client.post('/api/v1/invitations', data=json.dumps(new_invitation),
+                     headers=headers, content_type='application/json')
+    rv = client.get('/api/v1/invitations', headers=headers, content_type='application/json')
+    invitation_dict = json.loads(rv.data)
+    assert rv.status_code == http_status.HTTP_200_OK
+    assert invitation_dict['invitations']
+
+
+def test_get_invitations_by_id(client, jwt, session):  # pylint:disable=unused-argument
+    """Assert that an invitation can be retrieved."""
+    headers = factory_auth_header(jwt=jwt, claims=TEST_JWT_CLAIMS)
+    rv = client.post('/api/v1/users', headers=headers, content_type='application/json')
+    rv = client.post('/api/v1/orgs', data=json.dumps(TEST_ORG_INFO),
+                     headers=headers, content_type='application/json')
+    dictionary = json.loads(rv.data)
+    org_id = dictionary['id']
+    new_invitation = {
+        'recipientEmail': 'test@abc.com',
+        'sentDate': '2019-09-09',
+        'membership': [
+            {
+                'membershipType': 'MEMBER',
+                'orgId': str(org_id)
+            }
+        ]
+    }
+    rv = client.post('/api/v1/invitations', data=json.dumps(new_invitation),
+                     headers=headers, content_type='application/json')
+    invitation_dictionary = json.loads(rv.data)
+    invitation_id = invitation_dictionary['id']
+    rv = client.get('/api/v1/invitations/{}'.format(invitation_id), headers=headers, content_type='application/json')
+    assert rv.status_code == http_status.HTTP_200_OK
+
+
+def test_delete_invitation(client, jwt, session):  # pylint:disable=unused-argument
+    """Assert that an invitation can be deleted."""
+    headers = factory_auth_header(jwt=jwt, claims=TEST_JWT_CLAIMS)
+    rv = client.post('/api/v1/users', headers=headers, content_type='application/json')
+    rv = client.post('/api/v1/orgs', data=json.dumps(TEST_ORG_INFO),
+                     headers=headers, content_type='application/json')
+    dictionary = json.loads(rv.data)
+    org_id = dictionary['id']
+    new_invitation = {
+        'recipientEmail': 'test@abc.com',
+        'sentDate': '2019-09-09',
+        'membership': [
+            {
+                'membershipType': 'MEMBER',
+                'orgId': str(org_id)
+            }
+        ]
+    }
+    rv = client.post('/api/v1/invitations', data=json.dumps(new_invitation),
+                     headers=headers, content_type='application/json')
+    invitation_dictionary = json.loads(rv.data)
+    invitation_id = invitation_dictionary['id']
+    rv = client.delete('/api/v1/invitations/{}'.format(invitation_id), headers=headers, content_type='application/json')
+    assert rv.status_code == http_status.HTTP_200_OK
```

Comment:
We should also check here that the invitation has really been deleted by doing a GET and asserting an empty collection/response.

Reply:
Done